site/security.html

```diff
@@ -107,13 +107,13 @@ <h2>Discovering Security Vulnerabilities</h2>
           <ol>
             <li>Accessing, or attempting to access, accounts or data that does not belong to you or your Authorized Users,</li>
             <li>Any attempt to modify or destroy any data,</li>
-            <li>Executing, or attempting to execute, a denial of service attack, whether intentional or unintentional,</li>
+            <li>Executing, or attempting to execute, a denial-of-service attack, whether intentional or unintentional,</li>
             <li>Sending, or attempting to send, unsolicited or unauthorized email, spam or other forms of unsolicited messages,</li>
-            <li>Testing third party websites, applications or services that integrate with the FusionAuth Services, Cloud or self-hosted,</li>
+            <li>Testing third party websites, applications or services that integrate with the FusionAuth Cloud services or self-hosted services,</li>
             <li>Posting, transmitting, uploading, linking to, sending or storing malware, viruses or similar harmful software, or otherwise attempting to interrupt or degrade FusionAuth Cloud services or other FusionAuth web services including the FusionAuth website, and</li>
             <li>Any activity that violates any applicable law.</li>
           </ol>
-          <p>In the event that you do not have adequate clarity on the method of your discovery, please inquire before proceeding to ensure compliance.</p>
+          <p>Please take the time to read and research all of the above 7 points (for example Wikipedia's page on <a href="https://en.wikipedia.org/wiki/Denial-of-service_attack">denial-of-service attack</a> would be one place to start) and ensure you fully understand each. In the event that you do not have adequate clarity on the method of your discovery, please inquire before proceeding to ensure compliance.</p>
```

Comment:
I'm not against adding more context here, but I don't know that I like referring to wikipedia in context of what is legal or not legal to do in the process of discovering a vulnerability. If we think that there is too much ambiguity in the "do not DDoS us" statement - we could just say , please ensure you understand what DDoS means or something like that. 

Which is sort of what we say here:

> In the event that you do not have adequate clarity on the method of your discovery, please inquire before proceeding to ensure compliance.

If we are just trying to ensure they read this - (which I doubt anyone will - this content is really just a CYA addition) - but if that is the primary issue, we could keep the first sentence but shorten it a bit.

> Please take the time to read and understand each of the above points. In the event that you do not have adequate clarity on the method of your discovery, please inquire before proceeding to ensure compliance.

Reply:
That works for me.